client/shipping-label/views/index.js

```diff
@@ -25,10 +26,10 @@ const ShippingLabelRootView = ( props ) => {
 		<p className="wcc-metabox-button-container">
 			<GlobalNotices id="notices" notices={ notices.list } />
 			<PrintLabelDialog
-				{ ...( props.shippingLabel ) }
+				{ ...props.shippingLabel }
 				{ ...props }
 			/>
-			{ props.shippingLabel.success ? renderSuccessNotice() : renderPrintLabelFlow() }
+			{ props.shippingLabel.labels.purchased_time ? renderSuccessNotice() : renderPrintLabelFlow() }
```

Comment:
What does this do? Would the merchant ever see a success notice? Or would we ideally close the modal after the printing starts?